web/src/copy_and_paste.ts

```diff
@@ -552,6 +556,10 @@ export function paste_handler_converter(paste_html: string): string {
             // We convert single line code inside a code block to inline markdown code,
             // and the code for this is taken from upstream's `code` rule.
             if (!code.includes("\n")) {
+                // If the user just typed a backtick, then we don't add extra backticks.
+                if (compose_state.message_content().at(-1) === "`") {
```

Comment:
So this assume we're pasting at the end of the message, not where the cursor is. Probably some code in `compose_ui.ts` can help do this more correctly. We might also want to check if the immediately following character is a backtick too.

Also it's generally pretty easy to write tests for this file.

Reply:
> We might also want to check if the immediately following character is a backtick too.

I don't think we want to do this. I often type a single backtick and then paste something, without a backtick following my cursor.